src/DependencyInjection/Configuration.php

```diff
@@ -23,6 +23,7 @@ public function getConfigTreeBuilder(): TreeBuilder
                 ->booleanNode('autoinclude')->defaultTrue()->end()
                 ->scalarNode('base_path')->defaultValue('vendor/ace')->end()
                 ->booleanNode('debug')->defaultFalse()->end()
+                ->booleanNode('use_stimulus')->defaultFalse()->end()
```

Comment:
I think the default here should be null, not false, when use_stimulus is null, then we check for installed bundles, otherwise we respect user defined value.